automated install/uninstall.sh

```diff
@@ -206,22 +206,12 @@ removeNoPurge() {
 }
 
 ######### SCRIPT ###########
-if command -v vcgencmd &> /dev/null; then
-    echo -e "  ${INFO} All dependencies are safe to remove on Raspbian"
-else
-    echo -e "  ${INFO} Be sure to confirm if any dependencies should not be removed"
-fi
```

Comment:
Just change this part and address the other items in another PR as I've requested.

Reply:
Are you sure that you don't want a warning and don't want to change the default to "No", so that at least users hitting return on each question won't have everything purged? I through this was the absolute minimum we definitely agree in. If not, then please do that little change in a separate commit (much less work than reverting a PR and opening a new one) and I leave this PR until or in case we agree with a little more precaution for inexperienced users.

Reply:
I am sure that I want one change per PR. Makes it easier to revert if needed and only have one thing reverted. 

I have asked for changes as the maintainer of this repo. I'm not expecting pushback for my requests.